File: samples/cpp/tutorial_code/introduction/using_prebuilt_binaries/using_prebuilt_binaries.cpp
#include <opencv2/opencv.hpp>
#include <iostream>

int main(int argc, char** argv)
{
    cv::Mat image = cv::imread("lena.jpg", cv::IMREAD_COLOR);
    if (image.empty())
    {
        std::cerr << "Error: Could not open or find the image." << std::endl;
        return -1;
    }

    cv::Mat processed_image;
    cv::GaussianBlur(image, processed_image, cv::Size(7, 7), 1.5, 1.5);

    cv::imshow("Original", image);
    cv::imshow("Processed", processed_image);
    cv::waitKey(0);
    return 0;
}

File: samples/python/tutorial_code/introduction/using_prebuilt_binaries/using_prebuilt_binaries.py
import cv2
import sys

img = cv2.imread('lena.jpg')
if img is None:
    sys.exit("Error: image file missing or path invalid.")

gray_img = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
cv2.imshow('Color', img)
cv2.imshow('Grayscale', gray_img)
cv2.waitKey(0)
cv2.destroyAllWindows()
